.editorconfig

```
@@ -1,12 +1,16 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
 indent_size = 4
 trim_trailing_whitespace = true
 end_of_line = lf
 insert_final_newline = true
 
+[*.{json,jsonc,json5},.c8rc]
```

This doesn't seem to work. Maybe like this:

```suggestion
[{*.{json,jsonc,json5},.c8rc}]
```

---

It is working, tested by doing changes in the json and c8rc file and was able to see the spacing get updated as per the config during commit.

---

Also just tested, the one you suggested above, that is also working.